CRITICAL - 2022-12-20 03:11:02 --> Invalid file: default.php
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('default.php')
 2 SYSTEMPATH\View\View.php(229): CodeIgniter\View\View->render('default', [], true)
 3 SYSTEMPATH\Common.php(1134): CodeIgniter\View\View->render('home', [], true)
 4 APPPATH\Controllers\Home.php(9): view('home')
 5 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Home->index()
 6 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2022-12-20 04:06:21 --> Call to undefined function bsae_url()
in APPPATH\Views\layout\default.php on line 35.
 1 SYSTEMPATH\View\View.php(213): include()
 2 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\View\View.php(229): CodeIgniter\View\View->render('layout/default', [], true)
 4 SYSTEMPATH\Common.php(1134): CodeIgniter\View\View->render('home', [], true)
 5 APPPATH\Controllers\Home.php(9): view('home')
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
ERROR - 2022-12-20 07:21:52 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\yuknikah\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\yuknikah\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\xampp\htdocs\yuknikah\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\yuknikah\system\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\yuknikah\system\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
#4 C:\xampp\htdocs\yuknikah\app\Config\Routes.php(39): CodeIgniter\Database\Config::forge()
#5 C:\xampp\htdocs\yuknikah\system\CodeIgniter.php(868): CodeIgniter\CodeIgniter->Config\{closure}()
#6 C:\xampp\htdocs\yuknikah\system\CodeIgniter.php(469): CodeIgniter\CodeIgniter->startController()
#7 C:\xampp\htdocs\yuknikah\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\yuknikah\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\yuknikah\system\Commands\Server\rewrite.php(46): require_once('C:\\xampp\\htdocs...')
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\yuknikah\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\yuknikah\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\yuknikah\system\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\yuknikah\system\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
#3 C:\xampp\htdocs\yuknikah\app\Config\Routes.php(39): CodeIgniter\Database\Config::forge()
#4 C:\xampp\htdocs\yuknikah\system\CodeIgniter.php(868): CodeIgniter\CodeIgniter->Config\{closure}()
#5 C:\xampp\htdocs\yuknikah\system\CodeIgniter.php(469): CodeIgniter\CodeIgniter->startController()
#6 C:\xampp\htdocs\yuknikah\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\yuknikah\public\index.php(67): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\yuknikah\system\Commands\Server\rewrite.php(46): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2022-12-20 07:21:52 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\Config.php(109): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
 3 APPPATH\Config\Routes.php(39): CodeIgniter\Database\Config::forge()
 4 SYSTEMPATH\CodeIgniter.php(868): CodeIgniter\CodeIgniter->Config\{closure}()
 5 SYSTEMPATH\CodeIgniter.php(469): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
